File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/security/permissiongroup.ts

import { Permission } from './permission'
export class PermissionGroup {
    id: number;
    name: string;
    moduleId: number;
    permissions: Permission[];

    constructor(id: number,name: string,moduleId: number,permissions: Permission[]) {
        this.id = id;
        this.name = name,
        this.moduleId = moduleId,      
        this.permissions = permissions; 
    }
}

File: Service/Admin/RMA.Service.Admin.SecurityManager.Contracts/Entities/PermissionGroup.cs
using System.Collections.Generic;

namespace RMA.Service.Admin.SecurityManager.Contracts.Entities
{
    public class PermissionGroup
    {
        public int Id { get; set; }
        public int ModuleId { get; set; }
        public string Name { get; set; }
        public List<Permission> Permissions { get; set; }
    }
}
